### ISSUE TITLE
An e-commerce website wishes to analyze the per-day sales of their products. The products are tagged from a to z and A to Z. The daily sales are merged into a sequence of the tags of the products sold in the day. The company wishes to identify the products that are purchased only one time in the day.

Write an algorithm to find the count of the product tags that occur only once in the sales sequence.

### ISSUE THREAD

**Issue by UjjAwal021** [rutuja1302/WIPRO-NLTH#2]

**Comment by quatranon** [comment]
Code for the above:

#include <bits/stdc++.h>
using namespace std;
int main() {
    string s;
    cin>>s;

    int maxA = INT_MIN;
    int minA = INT_MAX;

    for(int i=0;i<s.length();i++){
        maxA = max(maxA,int(s[i]));
        minA = min(minA,int(s[i]));
    }
    cout<<maxA+minA;
}
